vscode/src/completions/client.ts

```diff
@@ -40,6 +42,8 @@ export function createClient(
             if (config.accessToken) {
                 headers.set('Authorization', `token ${config.accessToken}`)
             }
+            // c.f. https://github.com/microsoft/vscode/issues/173861
```

Comment:
Adding a one-liner explainer in addition to issue links is helpful to quickly understand the underlying problem without going to third-party sources.

```suggestion
            // Force HTTP connection reuse to reduce latency.
            // c.f. https://github.com/microsoft/vscode/issues/173861
```